tests/initests/invalidTranslations01.ini

```diff
@@ -1 +0,0 @@
-mautic.notification.notice.batch_deleted="%count% "upozornění bylo smazáno!"
```

Comment:
Shouldn't we use these examples of invalid ini files in our tests?

Reply:
Instead of physical files we have used it in the test code: https://github.com/ts-navghane/language-packer/blob/b6390be63897cb1a201f038a6b6d80254542415a/tests/Functional/Command/MauticLanguagePackerCommandTest.php#L271-L275